rllib/core/learner/tf/tf_learner.py

```diff
@@ -300,7 +300,7 @@ def _untraced_update(
         def helper(_batch):
             with tf.GradientTape(persistent=True) as tape:
                 fwd_out = self._module.forward_train(_batch)
-                loss_per_module = self.compute_loss(fwd_out=fwd_out, batch=_batch)
+                loss_per_module = self.compute_losses(fwd_out=fwd_out, batch=_batch)
```

Comment:
Do we even want this here and motivate users to write TF algorithms?

Reply:
Good point! But .... DreamerV3 :|